From 17950f63ac6a590a2f82eb9677ff65d304c8cb7e Mon Sep 17 00:00:00 2001
From: Yanjiang Jin <yanjiang.jin@windriver.com>
Date: Thu, 20 Mar 2014 15:59:26 +0800
Subject: [PATCH 049/122] cav-octeon3: set arch as octeon3

Use march=octeon3 other than march=octeon to support some cavium
special instructions, such as saa.

Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
Signed-off-by: Bin Jiang <bin.jiang@windriver.com>
---
 arch/mips/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/mips/Makefile b/arch/mips/Makefile
index 2408f15..def64ed 100644
--- a/arch/mips/Makefile
+++ b/arch/mips/Makefile
@@ -159,9 +159,9 @@ ifeq (,$(findstring march=octeon2, $(cflags-y)))
 cflags-y += -Wa,-march=octeon2
 endif
 else
-cflags-y += $(call cc-option,-march=octeon) -Wa,--trap
-ifeq (,$(findstring march=octeon, $(cflags-y)))
-cflags-y += -Wa,-march=octeon
+cflags-y += $(call cc-option,-march=octeon3) -Wa,--trap
+ifeq (,$(findstring march=octeon3, $(cflags-y)))
+cflags-y += -Wa,-march=octeon3
 endif
 endif
 endif
-- 
1.8.2.1

